src/lib/support/CodeUtils.h

```diff
@@ -257,7 +257,7 @@
         if (!::chip::ChipError::IsSuccess(__err))                                                                                  \
         {                                                                                                                          \
             ChipLogError(MOD, MSG ": %" CHIP_ERROR_FORMAT, ##__VA_ARGS__, __err.Format());                                         \
-            return _err;                                                                                                           \
+            return __err;                                                                                                          \
```

Comment:
While you're here, want to fix the typo in the "brief" bit?  Should be "log" instead of "lg".